client/allocrunner/taskrunner/logmon_hook.go

```diff
@@ -142,14 +149,27 @@ func (h *logmonHook) Prestart(ctx context.Context,
 	}
 }
 
-func (h *logmonHook) isLoggingDisabled() bool {
+func (h *logmonHook) isLoggingEnabled() bool {
+	if !h.config.enabled {
+		h.logger.Debug("log rotation is disabled by task")
```

Comment:
Is log rotation right here or does this relate to log collection? I can't figure out how this would look to an operator going through logs.